http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -89,14 +89,6 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
       _ <- F.raiseWhen(!hasPermission)(ForbiddenError(userInfo.userEmail))
 
       enableIntraNodeVisibility = req.labels.get(AOU_UI_LABEL).exists(x => x == "true")
-      // We only allow autopilot mode for members of CUSTOM_APP_USERS group
```

Comment:
removing this becuz this was added per @dinvlad 's request without workload separation

Reply:
So from now, we can try to create other Terra GKE apps using autopilot (mainly Cromwell and Galaxy), but we would need to do similar work on the app charts like what you had to do for the SAS and RStudio ones, correct?